packages/shared/src/components/filters/MyFeedHeading.tsx

```diff
@@ -82,7 +83,7 @@ function MyFeedHeading({
 
   const alertProps: Omit<AlertPointerProps, 'children'> = {
     offset: getOffset(),
-    isAlertDisabled,
+    isAlertDisabled: shouldHighlightFeedSettings ? isAlertDisabled : true,
```

Comment:
When do we set the isAlertDisabled?
Nothing really to add, just wanted to ensure we remove it correctly.

Reply:
https://github.com/dailydotdev/apps/blob/1af7586539ce01bd150cbe8f6c167d21f3fb6df9/packages/shared/src/components/layout/common.tsx#L82

It's related to some alert we apparently have for this, we could possibly remove completely as an alert, but it's used in one alert dot somewhere in mobile sidebar as well, so decided to leave as is.